lib/fabrik/DirectorManager.js

```diff
@@ -168,6 +169,7 @@ class DirectorManager extends BaseManager {
     const previousValues = _.get(params, 'previous_values');
     const action = _.isPlainObject(previousValues) ? 'update' : 'create';
     const opts = _.omit(params, 'previous_values');
+    args = _.assign(args, _.pick(params, 'parameters'));
```

Comment:
no need to pass the input username/password downstream via args object as its not going to be used. With credentials we must only pass them where required. So remove this set here and at line 192 & 193, directly work with params object instead.

Reply:
Done